src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -138,60 +149,107 @@ const TableHeader: React.FC<TableHeaderProps> = ({
-            {tableTitle}
-
-            {tableCapacity && (
-              <span className="private-table-seats-left">
-                {numberOfSeatsLeft} seats left
-              </span>
-            )}
-          </div>
-          {tableOfUser && tableOfUser.subtitle && (
-            <div className="private-table-subtitle">{tableOfUser.subtitle}</div>
-          )}
-        </div>
-        <div className="lock-button-container">
+    <div className="row table-header">
+      <div className="table-header__leave-table">
+        <button
+          type="button"
+          title={"Leave " + seatedAtTable}
+          className="table-header__leave-button"
+          id="leave-seat"
+          onClick={leaveSeat}
+        >
           <FontAwesomeIcon
-            className="lock-table-icon"
-            icon={isCurrentTableLocked ? faLock : faLockOpen}
-            size="sm"
+            className="table-header__leave-button--icon"
+            icon={faChevronLeft}
+            size="xs"
           />
-          <div className="lock-table-checbox-indication">
-            {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
+          Leave table
+        </button>
+      </div>
+
+      <div className="table-header__topic-info">
+        <div className="row table-header__topic">
+          <div>{tableTitle}</div>
+
+          <div className="table-header__edit-topic-button" onClick={show}>
+            <FontAwesomeIcon icon={faPen} />
           </div>
-          <label className="switch">
-            <input
-              type="checkbox"
-              className="switch-hidden-input"
-              checked={!!isCurrentTableLocked}
-              onChange={toggleIsCurrentTableLocked}
-            />
-            <span className="slider" />
-          </label>
         </div>
+
+        {tableCapacity && (
+          <span className="table-header__seats-left">
+            {numberOfSeatsLeft} seats left
+          </span>
+        )}
+
+        {tableOfUser && tableOfUser.subtitle && (
+          <label>{tableOfUser.subtitle}</label>
+        )}
+      </div>
+
+      <div className="table-header__lock-button">
+        <FontAwesomeIcon
+          className="table-header__lock-button--icon"
+          icon={isCurrentTableLocked ? faLock : faLockOpen}
+          size="sm"
+        />
+        <div className="table-header__lock-indication">
+          {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
+        </div>
+        <label className="switch table-header__lock-toggle">
+          <input
+            type="checkbox"
+            className="switch-hidden-input"
+            checked={!!isCurrentTableLocked}
+            onChange={toggleIsCurrentTableLocked}
+          />
+          <span className="slider" />
+        </label>
       </div>
+
+      <EditTableTitleModal
+        isShown={isShown}
+        title={tableTitle}
+        error={error}
+        onHide={hide}
+        onCancel={hide}
```

Comment:
De we intend/expect for these props to ever have different values passed to them?

Reply:
Chance are very low. This is a the component is specifically for `TableHeader.tsx` and it's in that location too.

Reply:
I'd probably just make them the same prop then.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a